src/hermezjs-typings.d.ts

```diff
@@ -395,7 +400,8 @@ declare module "@hermeznetwork/hermezjs/src/tx-utils" {
 
   function getMaxAmountFromMinimumFee(minimumFee: ScalarValue, balance: ScalarValue): ScalarValue;
 
-  // function getTransactionType();
+  function getTransactionType(transaction: Transaction): TxType;
```

Comment:
We already have an interface for transactions which is `Tx`. I know that in this case we just need the `to` from the `Tx` interface, so I would suggest to remove this new `Transaction` interface and change this signature to:

```ts
function getTransactionType(transaction: { to: Pick<Tx, "to"> }): TxType;
```

This is going to pick the same type of the `to` field in the already existing `Tx` interface without duplicating it.

Reply:
Sorry @adrifdez, forgot about it's holiday in your city today. I'll change it and merge it.